docs/deployment/config_descriptions.md

```diff
@@ -4,51 +4,93 @@ title: Configuration Descriptions
+| prefixNoDash | (required) | prefix with no dash
+| urs_url | uat.urs | urs url used for OAuth
+| api_backend_url | apigateway backend url | the API backend url
+| api_distribution_url | apigateway dist url | the API url used for file distribution
+| shared_data_bucket | cumulus-data-shared | the bucket has the shared data artifacts
+| system_bucket | (required) | the bucket used for storing deployment artifacts
+| buckets | N/A | Configuration of buckets with key, bucket name, and type (i.e. internal, public private)
+| cmr.username | devseed | the username used for posting metadata to CMR
+| cmr.provider | CUMULUS | the provider used for posting metadata to CMR
+| cmr.clientId | CUMULUS | the clientId used to authenticate with the CMR
+| cmr.password | (required) | the password used to authenticate with the CMR
+| ems.provider | CUMULUS | the provider used for sending reports to EMS
+| vpc.vpcId | (required if ecs is used) | the vpcId used with the deployment
+| vpc.subnets | (required) | the subnets used
+| vpc.cidrIp | (required) | the IPv4 CIDR of the VPC
+| ecs.amiid | ami-9eb4b1e5 | amiid of an optimized ecs instance (differnet for each region)
+| ecs.instanceType | (required) | the instance type of the ec2 machine used for running ecs tasks
+| ecs.volumeSize | 50 | the storage on ec2 instance running the ecs tasks
+| ecs.availabilityZone | us-east-1a | the availibity zone used for launching ec2 machines
+| ecs.minInstances | 1 | min number of ec2 instances to launch in an autoscaling group
+| ecs.desiredInstances | 1 | desired number of ec2 instances needed in an autoscaling group
+| ecs.maxInstances | 2 | max number of ec2 instances to launch in an autoscaling group
+| es.name | es5 | name of the elasticsearch cluster
+| es.elasticSearchMapping | 4 | version number of the elasticsearch mapping used
+| es.version | 5.3 | elasticsearch software version
+| es.instanceCount | 1 | number of elasticsearch nodes
+| es.instanceType | t2.small.elasticsearch | size of the ec2 instance used for the elasticsearch
+| es.volumeSize | 35 | the storage used in each elasticsearch node
+| sns.\<name\> | N/A | name of the sns topic
+| sns.\<name\>.subscriptions.lambda.endpoint | sns2elasticsearch | lambda function triggered for each message in the topic
+| apis.\<name\> | N/A | name of the apigateway application
+| apiStage | dev | stage name used for each api gateway deployment stage
+| dynamos.\<name\> | N/A | name of the dynamoDB table
+| dynamos.\<name\>.read | 5 | number of reads per second
+| dynamos.\<name\>.write | 1 | number of writes per second
+| dynamos.\<name\>.attributes | N/A | list of attributes
+| sqs.\<name\> | N/A | name of the queue
+| sqs.\<name\>.visibilityTimeout | 20 | # of seconds the message returns to the queue after it is read by a consumer
+| sqs.\<name\>.retry | 30 | number of time the message is returned to the queue before being discarded
+| sqs.\<name\>.consumer | N/A | list of lambda function queue consumers
```

Comment:
At least in our config, `consumer` is an array of objects. I guess that the above is an accurate description. I'm wondering if we should document the attributes of those objects though, since they impact how the consumer behaves